proxy/https.go

```diff
@@ -45,11 +45,12 @@ func (pxy *Proxy) handleHttps(lConn *net.TCPConn, exploit bool, initPkt *packet.
 	log.Debug("[HTTPS] Sent 200 Connection Estabalished to ", lConn.RemoteAddr())
 
 	// Read client hello
-	clientHello, err := ReadBytes(lConn)
+	hello, err := ReadClientHello(lConn)
```

Comment:
This is the best way. The client-hello message can also be fragmented, now we get it in its entirety